cypress/e2e/error-handling-spec.cy.js

```diff
@@ -0,0 +1,44 @@
+import gameOfThrones from '../fixtures/tvmaze.json';
+
+describe('Error handling', () => {
+    beforeEach(() => {
+        cy.visit('http://localhost:3000');
+    });
+
+    it('Should provide error message when show is not found', () => {
+        cy.intercept('GET', 'https://api.tvmaze.com/singlesearch/shows?q=fhfhfhf', {statusCode: 404})
+            .get('.search-input')
+            .type('fhfhfhf')
+            .get('.search-button')
+            .click()
+
+        cy.get('.error-message')
+            .contains('Error: Sorry, looks like we can\'t find this show')
+    });
+
+    it('Should provide error message when prescription can not be posted', () => {
+        cy.intercept('POST', 'https://tasterx-api.herokuapp.com/prescriptions/', {statusCode: 500})
+            .intercept('GET', 'https://www.tvmaze.com/shows/82/game-of-thrones', gameOfThrones)
+            .get('.search-input')
+            .type('game of thrones')
+            .get('.search-button')
+            .click()
+            .get('.text-area')
+            .type('Heck this aint gunna work.')
+            .get('.signature-input')
+            .type('Dr. Poo')
+            .get('.submit-button')
+            .click()
+
+        cy.get('.error-message')
+            .contains('Error: Sorry, there was an error posting your information.')
+        });
+    
+    it('Should navigate to prescription not found page if not prescription is found', () => {
```

Comment:
if prescription is NOT found*

Reply:
or I guess it could be "if no prescription is found"

Reply:
Updated this in the web editor so I can merge it